includes/header.php

```diff
@@ -115,6 +115,8 @@
 	$castPage = "";
 	$pollPage = "";
 	$pollArchivePage = "";
+	$adminMenu = "";
+	$avatarAdminPage = ""; $pollAdminPage = "";
```

Comment:
If we had a style guide, yeah. Otherwise, I was trying to make a page of var inits shorter without making my patch touch more than it needed to :)